src/routes/governance/vote-details.tsx

```diff
@@ -81,60 +61,6 @@ export const VoteDetails = ({ proposal }: VoteDetailsProps) => {
   }
   return (
     <section>
-      <h4 className="proposal__sub-title">{t("votes")}</h4>
-      <div>
-        <p className="proposal__set_to">
-          {t("setTo")}
-          <span className="proposal-toast__success-text">
-            <CurrentProposalStatus proposal={proposal} />
-          </span>
-          .&nbsp;
-          {proposal.state === ProposalState.Open ? daysLeft : null}
-        </p>
-        <table className="proposal-toast__table">
-          <thead>
-            <tr>
-              <th>{t("for")}</th>
-              <th>
-                <VoteProgress
```

Comment:
Is this component used anywhere else. Can we just delete it, we can always get it from version control